pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -268,10 +298,56 @@ spec:
               mountPath: /csi
           imagePullPolicy: {{ .ImagePullPolicy }}
         {{ end }}
+        {{ if .CSILogRotation }}
+        - args:
+          - |
+            echo "Starting the csi-logrotate-sidecar"
```

Comment:
Please move the script to Go code so we can avoid duplication of the script, and it can be formatted into the templates for the various pods.

Reply:
Done